src/server/config/cmds.go

```diff
@@ -374,13 +374,22 @@ func Cmds() []*cobra.Command {
 				return err
 			}
 
+			activeEnterpriseContext, _, err := cfg.ActiveEnterpriseContext(true)
+			if err != nil {
+				return err
+			}
+
 			fmt.Println(listContextHeader)
 			for _, key := range keys {
+				var activeMarker string
 				if key == activeContext {
-					fmt.Printf("*\t%s\n", key)
-				} else {
-					fmt.Printf("\t%s\n", key)
+					activeMarker = "*"
+				}
+				var activeEnterpriseMarker string
+				if key == activeEnterpriseContext {
+					activeEnterpriseMarker = "E"
```

Comment:
IMO if the enterprise context == the active context, we should _only_ print the `*`. Otherwise I think it's going to be confusing for users who aren't using enterprise.